src/deploy-to-network.ts

```diff
@@ -0,0 +1,162 @@
+
+async function askForBigNumber(numberUsage: string): Promise<BigNumber> {
+  for (;;) {
+    const bigNumber = await askFor("Enter " + numberUsage + ":");
+    try {
+      return BigNumber.from(bigNumber);
+    } catch (e) {
+      console.log("This is not a valid number");
+    }
+  }
+}
+
+async function askYesNo(question: string): Promise<boolean> {
+  for (;;) {
+    switch (await askFor(question + " (yes/no):")) {
+      case "y":
+      case "yes":
+        return true;
+      case "n":
+      case "no":
+        return false;
+    }
+  }
+}
+
+async function askFor(question: string): Promise<string> {
+  console.log(question);
+  const rl = readline.createInterface({ input: process.stdin });
+  const next = await rl[Symbol.asyncIterator]().next();
+  rl.close();
+  return next.value as string;
+}
+
+function printDeployed(name: string, contract: Contract): void {
+  console.log("Deployed", name, "contract to address", contract.address);
+}
+
+export async function testEns(): Promise<void> {
+  console.log("The wallet address will become the owner of the 'eth' domain");
+  const provider = await walletConnectProvider();
+  const signer = provider.getSigner();
+  const signerAddr = await signer.getAddress();
+  const ens = await deploy.deployEns(signer);
+  printDeployed("ENS", ens);
+  await deploy.setSubnodeOwner(ens, "", "eth", signerAddr);
+  console.log("Registered domain 'eth' for", signerAddr);
+  await provider.close();
+}
+
+export async function phase0(): Promise<void> {
+  console.log("The wallet address must own the 'eth' domain in the ENS");
+  const provider = await walletConnectProvider();
+  const signer = provider.getSigner();
+  const govGuardian = await askForAddress("of the governor guardian");
+  const tokensHolder = await askForAddress("to hold all the Radicle Tokens");
+  const ensAddr = await askForAddress("of the ENS");
+
+  const token = await deploy.deployRadicleToken(signer, tokensHolder);
+  printDeployed("Radicle Token", token);
+
+  const govAddr = utils.getContractAddress({
+    from: await signer.getAddress(),
+    nonce: 1 + (await signer.getTransactionCount()),
+  });
+  const delay = 2 * 60 * 60 * 24;
+  const timelock = await deploy.deployTimelock(signer, govAddr, delay);
+  printDeployed("timelock", timelock);
+
+  const gov = await deploy.deployGovernance(
+    signer,
+    timelock.address,
+    token.address,
+    govGuardian
+  );
+  printDeployed("governance", gov);
+
+  const treasury = await deploy.deployTreasury(signer, govAddr);
+  printDeployed("treasury", treasury);
+
+  const ens = ENS__factory.connect(ensAddr, signer);
+  const registrar = await deploy.deployRegistrar(
+    signer,
+    constants.AddressZero, // oracle not used yet
+    constants.AddressZero, // exchange not used yet
+    token.address,
+    ens,
+    "eth",
+    "radicle",
+    govAddr
+  );
+  printDeployed("registrar", registrar);
+
+  await provider.close();
+}
+
+export async function vestedTokens(): Promise<void> {
```

Comment:
```suggestion
export async function vestingTokens(): Promise<void> {
```
The yarn task should be renamed too.

Reply:
Fixed